google/cloud/storage/async/client_test.cc

```diff
@@ -92,20 +92,24 @@ TEST(AsyncClient, InsertObject) {
   EXPECT_THAT(response, IsOkAndHolds(TestObject()));
 }
 
-TEST(AsyncClient, ReadObject) {
+TEST(AsyncClient, ReadObject1) {
+  auto constexpr kExpectedRequest = R"pb(
+    bucket: "projects/_/buckets/test-bucket"
+    object: "test-object"
+  )pb";
   auto mock = std::make_shared<MockAsyncConnection>();
   EXPECT_CALL(*mock, options)
       .WillRepeatedly(
           Return(Options{}.set<TestOption<0>>("O0").set<TestOption<1>>("O1")));
 
   EXPECT_CALL(*mock, ReadObject)
-      .WillOnce([](AsyncConnection::ReadObjectParams const& p) {
+      .WillOnce([&](AsyncConnection::ReadObjectParams const& p) {
```

Comment:
:smile:

Reply:
I try to learn. I also fail to learn, but I try to.